src/avrdude.1

```diff
@@ -1535,6 +1535,19 @@ alone.
 .It Ar help
 Show help menu and exit.
 .El
+.It Ar PICkit 4
+.It Ar MPLAB SNAP
+.Bl -tag -offset indent -width indent
+.It Ar mode=avr,pic
+The PICkit 4 and MPLAB SNAP programmer can only be utilized by Avrdude when in
```

Comment:
Should document that avrdude terminates after `-xmode=avr` or `-xmode=pic`. COuld say
```
Switch programmer to AVR or PIC mode, then exit: the PICkit 4 and MPLAB SNAP programmer can only be utilised by
.Nm
when in AVR mode.
```